android/build.gradle

```diff
@@ -46,4 +47,10 @@ android {
 dependencies {
     implementation 'com.facebook.react:react-native:+'
     implementation 'com.brightcove.player:exoplayer2:6.8.1'
+
+    // IMA SDK
+    implementation 'com.google.ads.interactivemedia.v3:interactivemedia:3.16.0'
+    implementation 'com.google.android.gms:play-services-ads-identifier:17.0.0'
+    implementation "androidx.core:core-ktx:1.0.1"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
```

Comment:
```
def safeExtGet(prop, fallback) {
    rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
}

 implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:${safeExtGet('kotlin_version', '1.3.50')}"

```

Please use this, so that we can maintain same kotlin version for all libraries in our project